website/source/docs/configuration/server.html.md

```diff
@@ -77,6 +77,10 @@ server {
   terminal state before it is garbage collected and purged from the system. This
   is specified using a label suffix like "30s" or "1h".
 
+- `job_gc_interval` `(string: "5m")` - Specifies the interval between the job
+  garbage collection loops. This is specified using a label suffix like "30s"
+  or "3m".
```

Comment:
It would be nice to describe _why_ someone might want to change this parameter as it's a far more niche tunable than `job_gc_threshold`.

Perhaps something like:

```
- `job_gc_interval` `(string: "5m")` - Specifies the interval between the job
  garbage collections. Only jobs who have been terminal for at least
  `job_gc_threshold` will be collected. Lowering the interval will perform more
  frequent but smaller collections. Raising the interval will perform collections
  less frequently but collect more jobs at a time. This is specified using a
  label suffix like "30s" or "3m".
```

But that's a bit wordy, so I don't know. Not a blocker, but if you or anyone else wants to sneak a little more detail into the docs I think that would be helpful.

Reply:
I've rebased and added some extra info.

Reply:
Perfect! Thanks! 0.9.4 final should go out next week and then we'll merge.